Sources_v3/APIClient/Endpoints/Payloads/DevicePayloads.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+struct DevicePayload: Decodable, Equatable {
+    private enum CodingKeys: String, CodingKey {
+        case id
+        case createdAt = "created_at"
+    }
+    
+    /// Device identifier.
+    let id: String
+    /// Date the device was created for the user.
+    let createdAt: Date?
+    
+    init(from decoder: Decoder) throws {
```

Comment:
It doesn't work out of the box? 🤔

Reply:
Overlook :D Fixed in 46f926451006e1b0be9125afc4109583cef1a9f7